azure/test/blobs_logs_monitoring.test.js

```diff
@@ -615,3 +615,25 @@ describe('Batching', function() {
         });
     });
 });
+
+describe('HTTPClient', function() {
```

Comment:
could you add this test and the associated change to the other forwarder as well please?

Reply:
https://github.com/DataDog/datadog-serverless-functions/pull/460
looks like it was fixed 3 years ago there, ill add the unit test for posterity's sake